api/base/api_globals.py

```diff
@@ -0,0 +1,10 @@
+"""Global variables related to Django requests
+
+Made available in a separate file so as to be importable by Flask code, before (or in the absence of) Django
+"""
+import threading
+
+api_globals = threading.local()
```

Comment:
Would it be beneficial to use werkzeug's LocalProxy instead of using the `threading` module directly?